packages/exchange/src/translators/reactjs/hooks/useCreateReferralLink.tsx

```diff
@@ -0,0 +1,52 @@
+import {Exchange} from '@yoroi/types'
+import {UseQueryOptions, useQuery} from 'react-query'
+
+export const useCreateReferralLink = (
+  {
+    getBaseUrl,
```

Comment:
I'd say it should receive the provider and figure it out the URL by itself, the client should not know about getBaseUrl, the client should know only which provider was selected and if it is production.